codice/ddf
# DDF-05094 Remove the Handlebars WFS feature transformer's coordinate order configuration

#### What does this PR do?
Removes the Handlebars WFS feature transformer's coordinate order configuration. The transformer has access to a WfsMetadata object that contains the coordinate order of the WFS source corresponding to the feature it is about to transform.

#### Who is reviewing it? 
@dcruver 
@leo-sakh 
@gjvera 
@bennuttle 

#### Select relevant component teams: 
@codice/ogc 

#### Ask 2 committers to review/merge the PR and tag them here.
@jlcsmith
@troymohl

#### How should this be tested?
1. In the Admin UI, open the Spatial app.
2. Click `WFS Feature to Metacard Templated Mapping`.
3. For `Feature Type`, enter `{http://stategeothermaldata.org/uri-gin/aasg/xmlschema/welllog/0.8}AZWellLogs`.
4. For `Attribute Mappings`, add `{"attributeName":"location","featureName":"Shape","template":"{{Shape}}"}`.
5. Click `Save changes`.
6. Create a WFS 1.1.0 source with the URL http://services.azgs.az.gov/ArcGIS/services/aasggeothermal/AZWellLogs/MapServer/WFSServer
7. Open Intrigue and swap the 3D map for the 2D map.
8. Run a * query against the WFS source.
9. Verify the results are displayed on the map in Arizona.
10. Go back to the Admin UI, edit the source, and change the coordinate order to Lon/Lat.
11. Rerun the query in Intrigue and verify the results are displayed below the map.
a. Wait for the results from the source. The results from the cache will come back first and will still be displayed in Arizona.
12. Change the coordinate order back to Lat/Lon, rerun the query and verify the results are back in Arizona.

#### What are the relevant tickets?
Fixes: #5094 

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
